<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DHT Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="peerTab">
       <attribute name="title">
        <string>Peers</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="peerList">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QPushButton" name="searchButton">
          <property name="toolTip">
           <string>Search for hash</string>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="searchInput">
          <property name="font">
           <font>
            <family>Bitstream Vera Sans Mono</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2" colspan="3">
         <layout class="QHBoxLayout" name="resultLayout">
          <item>
           <layout class="QVBoxLayout" name="controlLayout">
            <item>
             <widget class="QPushButton" name="refreshButton">
              <property name="toolTip">
               <string>Refresh results</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme=":/icons/refresh.svg"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="toolTip">
               <string>Delete selected search results</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme=":/icons/clear.svg"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="copyButton">
              <property name="toolTip">
               <string>Copy results to clipboard</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme=":/icons/document.svg"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="searchList">
            <property name="font">
             <font>
              <family>Bitstream Vera Sans Mono</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="searchResults"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="searchTypeCombo">
          <item>
           <property name="text">
            <string>Hash</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="announceTab">
       <attribute name="title">
        <string>Announce</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionHide"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>&amp;Hide</string>
   </property>
  </action>
  <action name="actionStartStop">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
